.github/workflows/main.yml

```
@@ -13,72 +13,72 @@
+        uses: actions/cache@v4
+        with:
+          path: spec/dummy/vendor/bundle
+          key: dummy-app-gem-cache-${{ hashFiles('react_on_rails.gemspec') }}-${{ hashFiles('Gemfile.development_dependencies') }}-${{ matrix.versions }}
+      - name: Install Ruby Gems for dummy app
+        run: cd spec/dummy && bundle lock --add-platform 'x86_64-linux' && bundle check --path=vendor/bundle || bundle _2.5.9_ install --path=vendor/bundle --jobs=4 --retry=3
+      - name: generate file system-based packs
+        run: cd spec/dummy && RAILS_ENV=test bundle exec rake react_on_rails:generate_packs
+      - name: Build test bundles for dummy app
+        run: cd spec/dummy && rm -rf public/webpack/test && yarn build:rescript && RAILS_ENV=test NODE_ENV=test bin/${{ matrix.versions == 'oldest' && 'web' || 'shaka' }}packer
+      - id: get-sha
+        run: echo "::set-output name=sha::$(git rev-parse HEAD)"
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)